### ISSUE TITLE
Downloadable binaries version ignoring UseSystemTextJson

### ISSUE THREAD

**Issue by mgkeen** [zijianhuang/openapiclientgen#65]
I downloaded the [OpenApiClientGen310_P.zip](https://github.com/zijianhuang/openapiclientgen/releases/download/v3.1_PluginUpdated/OpenApiClientGen310_P.zip) version of the binary. When I ran that against a `CodeGenSettings.json` file which had `"UseSystemTextJson": true`, it still generated with newtonsoft.

I cloned the repo so I could reproduce and fix whatever bug was going on, but it was working fine when running from the solution with the same settings file and schema.

The settings.json file for reference:

```
{
  "ClientNamespace": "Readacted",
  "ClientLibraryProjectFolderName": ".",
  "ContainerClassName": "RedactedClient",
  "ClientLibraryFileName": "Redacted.cs",
  "ActionNameStrategy": 4,
  "UseEnsureSuccessStatusCodeEx": true,
  "DecorateDataModelWithDataContract": true,
  "DataContractNamespace": "https://example.com/v1",
  "DataAnnotationsEnabled": true,
  "DataAnnotationsToComments": true,
  "HandleHttpRequestHeaders": true,
  "UseNullableQuestionMark": true,
  "UseSystemTextJson": true,
  "UseNullableReferenceType": true,
  "ArrayAs": 5,
  "EnumToString": true
}
```

**Comment by ZijianFLG** [comment]
@mgkeen , I have just tested with what in https://github.com/zijianhuang/openapiclientgen/tree/master/Tests/DemoClientApiTextJson

It works as expected.

**Comment by ZijianFLG** [comment]
then tested with your codegen settings.json, all good with System.Text.Json.

**Comment by zijianhuang** [comment]
then use the Downloadable binaries, all OK.

**Comment by zijianhuang** [comment]
@mgkeen , another possible reason could be that your yaml file may trigger some bugs in the code gen, would you mind to share it?

**Comment by zijianhuang** [comment]
5 out of over 2000 definition files have problems when using UseSystemTextJson. Fixed. Released v3.1.1
